lib/ExpensiMark.js

```diff
@@ -722,4 +721,31 @@ export default class ExpensiMark {
         // If there are any tags left in the stack, they're unclosed
         return tagStack.length !== 0;
     }
+
+    /**
+     * @param {String} comment
+     * @returns {Array}
+     */
+    extractLinksInMarkdownComment(comment) {
+        const regex = new RegExp(MARKDOWN_LINK_REGEX, 'gi');
```

Comment:
Let's declare this at the top of the file.

Reply:
Done